main/src/main/scala/org/clulab/numeric/mentions/package.scala

```diff
@@ -369,11 +383,12 @@ package object mentions {
       if(seasonNorm.isEmpty)
         throw new RuntimeException(s"ERROR: could not find argument season in mention ${m.raw.mkString(" ")}!")
```

Comment:
It doesn't especially matter in an error string, but I wonder why m.text is not used.  Perhaps it didn't exist at some point.

Reply:
...or maybe being raw was very important.

Reply:
To be honest, I did not think much about it. I just followed the same code as the functions above.